src/gui_elements.py

```diff
@@ -172,12 +176,11 @@ def hover_click(self, event: pygame.event) -> None:
         :param event: Действия пользователя
         """
         collide = super().check_collision()
+        # Проверка на коллизию мышки с кнопкой.
         if collide:
             self.button_texture = self.hover_texture
             if event.type == pygame.MOUSEBUTTONDOWN:
                 self.button_texture = self.click_texture
```

Comment:
Наслаждайся (НЕ) своим решением элементарной проблемы.
```suggestion
            if event.type == pygame.MOUSEBUTTONDOWN:
                self.button_texture = self.click_texture
            else:
                self.button_texture = self.hover_texture
```